src/sentry/models/apitoken.py

```diff
@@ -104,10 +240,16 @@ def get_allowed_origins(self):
         return ()
 
     def refresh(self, expires_at=None):
+        if self.token_type == AuthTokenType.USER:
+            raise NotImplementedError("User auth tokens do not support refreshing the token")
```

Comment:
Should this be a NotImplementedError then?
Or should we make this like a `raise NotSupported` instead?

Reply:
Ah, yeah, I think that's a better idea.

The [Python docs](https://docs.python.org/3/library/exceptions.html#NotImplementedError) seem to agree as well.

> It should not be used to indicate that an operator or method is not meant to be supported at all – in that case either leave the operator / method undefined or, if a subclass, set it to [None](https://docs.python.org/3/library/constants.html#None).